libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.styles.ts

```diff
@@ -53,6 +54,9 @@ export const modelOverviewStyles: () => IProcessedStyleSet<IModelOverviewStyles>
         padding: "0 40px 32px 40px"
       },
       selections: flexLgDown,
+      slider: {
+        width: "250px"
```

Comment:
from the screenshot in the description, the slider bar looks too small to me. Is it by design that we keep all these widgets in one line? Can we move some to the second line?

Reply:
Great question, the screenshot is old, the slider is a quite bigger now.